<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMoneyDialog</class>
 <widget class="QDialog" name="AddMoneyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(102, 16, 53);
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>20</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: '#ebebeb';
 background: '#75103c';
 color: '#e8e8e8';
 border: 4px solid '#1a1818';
 border-radius: 15px;
}

QLabel::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Add Money</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="phraseLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Add money to your bank account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 16pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}

QLineEdit::hover {
border: 4px solid '#000000';
}

QLineEdit::focus {
border: 4px solid '#000000';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>295</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Save | Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
